presto-docs/src/main/sphinx/release/release-0.288.rst

```diff
@@ -0,0 +1,91 @@
+* Fix an error for some queries using a mix of joins and semi-joins when grouped execution is enabled. :pr:`22538`
+* Fix :func:`!array_join` to not add a trailing delimeter when the last element in the array is NULL. :pr:`22652`
+* Fix cast of NaN and Infinity from DOUBLE or REAL to  BIGINT, INTEGER, SMALLINT, and TINYINT. It will now return an exception with the INVALID_CAST_ARGUMENT error code. Previously it would return zero. :pr:`22917`
+* Fix compilation error for queries with lambda in aggregation function. :pr:`22539`
+* Fix incorrect behaviors when defining duplicate field names in RowType and throw exception uniformly. :pr:`22618`
+* Fix wrong results for :func:`!regr_r2`. :pr:`22611`
+* Fix the latency regression for queries with large IN clause. :pr:`22661`
+* Fix wrong results when queries using materialized CTEs have multiple common filters pushed into the CTE. :pr:`22700`
+* Improve :doc:`/sql/explain-analyze` statement to support a ``format`` argument with values of ``<TEXT|JSON>``. :pr:`22733`
+* Improve README.md and CONTRIBUTING.md. :pr:`22918`
+* Improve configuring worker threads relative to core count by setting the ``task.max-worker-threads`` configuration property to ``<multiplier>C``. For example, setting the property to ``2C`` configures the worker thread pool to create up to twice as many threads as there are cores available on a machine. :pr:`22809`
+* Improve logging for RowExpressionRewriteRuleSet and StatsRecordingPlanOptimizer optimizers to include more information. :pr:`22765`
+* Improve session property ``property-use_broadcast_when_buildsize_small_probeside_unknown`` to do broadcast join when probe side size is unknown and build side estimation from HBO is small. :pr:`22681`
+* Improve the estimation stats recorded during query optimization. :pr:`22769`
+* Improve Presto C++ documentation. :pr:`22717`
+* Improve error code for cast from DOUBLE or REAL to BIGINT, INTEGER, SMALLINT or TINYINT for out of range values from NUMERIC_VALUE_OUT_OF_RANGE to INVALID_CAST_ARGUMENT. :pr:`22917`
+* Improve handling of floating point numbers in Presto to consistently treat NaNs as larger than any other number and equal to itself. It also changes the handling of positive and negative zero to always be considered equal to each other. Read more here: https://github.com/prestodb/rfcs/blob/main/RFC-0001-nan-definition.md. The new nan behavior can be disabled by setting the configuration property use-new-nan-definition to false. This configuration property is intended to be temporary to ease migration in the short term, and will be removed in a future release. :pr:`22386`
+* Improve the performance of reading common table expressions (CTE). :pr:`22478`
+* Improve join performance by prefiltering the build side with distinct keys from the probe side. This can be enabled with the ``join_prefilter_build_side `` session property. :pr:`22667`
```

Comment:
```suggestion
* Improve join performance by prefiltering the build side with distinct keys from the probe side. This can be enabled with the ``join_prefilter_build_side`` session property. :pr:`22667`
```
The space at the end of ``join_prefilter_build_side `` causes the following doc build warning

`/Users/steveburnett/Documents/GitHub/presto/presto-docs/src/main/sphinx/release/release-0.288.rst:39: WARNING: Inline literal start-string without end-string.`
Deleted the space locally then did a new doc build to verify the warning was gone.